Marketplace/Middlewares/soloAdmin.js

```diff
@@ -0,0 +1,6 @@
+module.exports = (req, res, next) => {
+	if (!req.session.usuarioLogeado && req.session.usuarioLogeado.rol_id !=1 ) {
```

Comment:
Esta mal la logica aca
```suggestion
	if (req.session.usuarioLogeado && req.session.usuarioLogeado.rol_id !=1 ) {
```

Reply:
Gracias Christian. Las preguntas están bien. Pero es cierto que conviene subdividirlas, con diferentes soluciones. Ya está hecho.